client/shipping-label/components/label-purchase-modal/index.js

```diff
@@ -9,13 +9,14 @@ import { translate as __ } from 'i18n-calypso';
  */
 import Modal from 'components/modal';
 import ActionButtons from 'components/action-buttons';
-import Spinner from 'components/spinner';
+import LoadingSpinner from 'components/loading-spinner';
 import getPDFSupport from 'lib/utils/pdf-support';
-import AddressStep from './steps/address';
-import PackagesStep from './steps/packages';
-import RatesStep from './steps/rates';
+import AddressStep from './address-step';
+import PackagesStep from './packages-step';
+import RatesStep from './rates-step';
 import Sidebar from './sidebar';
 import { getRatesTotal } from 'shipping-label/state/selectors/rates';
+import FormSectionHeading from 'components/forms/form-section-heading';
 
 const PrintLabelDialog = ( props ) => {
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-services/pull/1051/commits/9e7d3bc41612be97fdab88fab7e181de448a06bf